torchmetrics/metric.py

```diff
@@ -40,6 +40,13 @@ def jit_distributed_available() -> bool:
     return torch.distributed.is_available() and torch.distributed.is_initialized()
 
 
+def is_overridden(method: str, instance: object, parent: object) -> bool:
```

Comment:
In PL we have a more complex implementation that also supports mocks and partials. Considerin copying it: https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/utilities/model_helpers.py#L21

Reply:
thanks :]